src/man/sssd.conf.5.xml

```diff
@@ -790,6 +788,32 @@
                         </para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term>refresh_max_random_offset (integer)</term>
+                    <listitem>
+                        <para>
+                            When SSSD is in offline mode it keeps probing
+                            backend servers in specified time intervals:
+                        </para>
+                        <para>
+                            delay = (delay * 2) % offline_timeout_max + (rand() % refresh_max_random_offset)
+                        </para>
+                        <para>
+                            This parameter controls the value of the random offset
+                            used for above equation. Final random_offset value
```

Comment:
Little correction: `for the above equation`.

Reply:
Fixed